packages/ruby/lib/http_response.rb

```diff
@@ -7,4 +8,32 @@ class HttpResponse < SimpleDelegator
   def self.from_parts(status, headers, body)
     new(Rack::Response.new(body, status, headers))
   end
+
+  def body
+    if super.respond_to?(:rewind)
+      super.rewind
+      read_body = super.each.reduce("", :+)
+      super.rewind
+
+      read_body
+    else
+      super.each.reduce("", :+)
+    end
+  end
```

Comment:
When using simple delegator, I think it's more common to use `__getobj__` than `super`. If you want you could pull out a private method:

```ruby
def rack_request
  __getobj__
end
```

or you could add line  in the body function like:

```ruby
raw_body = __getobj__.body
```